Install operator through marketplace documentation (#247)
rancher/elemental-docs: docs/quickstart-ui.md

If the `elemental` plugin is listed and the status stays at `Installing...`, refresh the page. The `elemental` plugin will display correctly.
:::

Once the `elemental` plugin installed, you can see the `OS Management` option in the Rancher Manager menu, refresh the page if you do not see it.

![Rancher Manager OS Management menu](images/quickstart-ui-elemental-plugin-menu.png)

## Install the elemental operator

:::note

> We will guide you to install the operator through the Elemental UI but it can also be installed directly from the Marketplace.

Reviewer: ```suggestion
The following guide will show you how to install the operator through the Elemental UI. But you can also install it directly from the Marketplace.
```

:::

Click on the OS Management button in the navigation menu.

If the operator is not already installed, the elemental ui will let you deploy it by clicking on the `Install Elemental Operator` button:
![Button to deploy elemental operator](images/quickstart-ui-extension-operator-button.png)

It will redirect you to the Rancher Marketplace to install the operator.

Click on the `Next` button:
![Install Elemental operator screenshot 1](images/quickstart-ui-extension-operator-install-1.png)

In this screen, you can customize or use the default values, click on `Install` to continue:
![Install Elemental operator screenshot 2](images/quickstart-ui-extension-operator-install-2.png)

You should see `elemental-operator-crds`and `elemental-operator` deployed in the `cattle-elemental-system` namespace:
![Install Elemental operator screenshot 3](images/quickstart-ui-extension-operator-install-3.png)

> :::note
> If you do not see them, make sure to select the correct namespace at the top of the page.
> :::

Reviewer: ```suggestion
:::warning
If you do not see them, make sure to select the correct namespace at the top of the page.
:::
```
